tests/test_app/src/Model/Endpoint/SearchEndpoint.php

```diff
@@ -1,14 +0,0 @@
-<?php
-declare(strict_types=1);
-
-namespace Crud\Test\App\Model\Endpoint;
-
-use Muffin\Webservice\Model\Endpoint;
```

Comment:
Revert this deleting, it's for the webservice plugin which might be upgrade for Cake5 someday.